### PR TITLE
Load all fixtures before rspec test suite is run.

### PR DESCRIPTION
**Pull request by HalahAb**

### CODE REVIEW

**Comment by HalahAb** [comment]
Resolves #59

File: spec/requests/api/c/rss_search_spec.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
   let(:affiliate) { affiliates(:basic_affiliate) }
 
-  before(:all) do
+  before(:each) do
```
Comment (MothOnMars) [inline spec/requests/api/c/rss_search_spec.rb]: you can simplify this as `before do` (which is equivalent to `before(:each)`

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -51,11 +51,11 @@
     c.syntax = [:expect]
   end
 
-  # This prevents affiliate-related tests from failing with an empty test db
+  # This prevents fixture related fails with an empty test db
```
Comment (MothOnMars) [inline spec/spec_helper.rb]: @HalahAb , let's simplify this entire comment to be:
`# Create all fixtures in spec/fixtures/*.yml for all tests in alphabetical order.`

Comment (HalahAb) [reply spec/spec_helper.rb]: should I get rid of the comment about the language fixtures as well? or should we keep it for future reference?

**Pull request merged**